api/alerts.go

```diff
@@ -108,14 +108,13 @@ func (a *Alerts) Update(viewName string, newAlert *Alert) (*Alert, error) {
 	return &alert, nil
 }
 
-func (a *Alerts) Add(viewName string, newAlert *Alert) (*Alert, error) {
+func (a *Alerts) Add(viewName string, newAlert *Alert, runAsUserID string) error {
```

Comment:
The `Update` function should also be updated to allow specifying the run-as-user.

Reply:
Done!